Podfile.lock

```diff
@@ -1,41 +1,44 @@
 PODS:
   - Crashlytics (3.8.3):
     - Fabric (~> 1.6.3)
+  - DZNEmptyDataSet (1.8.1)
   - Fabric (1.6.11)
   - MBProgressHUD (1.0.0)
-  - measurement_kit (0.4.0-beta.4)
+  - measurement_kit (0.4.0)
```

Comment:
We should commit it because it tells `pod` the version and hash of the commit used (this is something I wasn't aware of and that you told me :-)